lib/double_entry/transfer.rb

```diff
@@ -115,8 +115,15 @@ def create_lines(amount, code, detail, from_account, to_account)
 
     def create_line_metadata(credit, debit, metadata)
       metadata.each_pair do |key, value|
-        LineMetadata.create!(:line => credit, :key => key, :value => value)
-        LineMetadata.create!(:line => debit, :key => key, :value => value)
+        if value.is_a?(Array)
+          value.each do |each_value|
+            LineMetadata.create!(:line => credit, :key => key, :value => each_value)
+            LineMetadata.create!(:line => debit, :key => key, :value => each_value)
+          end
+        else
+          LineMetadata.create!(:line => credit, :key => key, :value => value)
+          LineMetadata.create!(:line => debit, :key => key, :value => value)
+        end
```

Comment:
Heya, @spickermann 
TIL, that is a great trick. I didn't know Ruby allows this coercion.